### PR TITLE
Migrate Doctrine mappings to XML.

### LINKED ISSUE #37: Convert doctrine mapping from yml to XML
I've noticed this link some days ago :
https://github.com/doctrine/DoctrineBundle/issues/776

Doctrine bundle will not suport yml mapping for mapping. It would be nice if this boilerplate could migrate.

`php bin/console doctrine:mapping:convert xml ./config/mapping/orm`
should work

### PR DESCRIPTION
**Pull request by Lutacon**
Fixes #37.

### CODE REVIEW

File: src/UI/Http/Rest/Controller/Event/GetEventsController.php [hunk 1]
```diff
@@ -7,7 +7,7 @@
 use App\Application\Query\Collection;
 use App\Application\Query\Event\GetEvents\GetEventsQuery;
 use App\UI\Http\Rest\Controller\QueryController;
-use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Component\Routing\Annotation\Route;
```
Comment (jorge07) [inline src/UI/Http/Rest/Controller/Event/GetEventsController.php]: Whats the reason behind this?

File: config/packages/framework_extra.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+sensio_framework_extra:
```
Comment (jorge07) [inline config/packages/framework_extra.yml]: Can this be removed?

File: config/mapping/orm/domain/User.ValueObject.Auth.Credentials.orm.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping"
+                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                  xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
+    <embeddable name="App\Domain\User\ValueObject\Auth\Credentials" table="credentials">
```
Comment (jorge07) [inline config/mapping/orm/domain/User.ValueObject.Auth.Credentials.orm.xml]: is `table="credentials"` needed?

File: config/mapping/orm/projections/User.Query.UserView.orm.xml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping"
+                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                  xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
+    <entity name="App\Infrastructure\User\Query\UserView" table="users">
+        <id name="uuid" type="uuid_binary" column="uuid"/>
+        <embedded name="credentials" class="App\Domain\User\ValueObject\Auth\Credentials"/>
+    </entity>
+</doctrine-mapping>
```
Comment (jorge07) [inline config/mapping/orm/projections/User.Query.UserView.orm.xml]: missing nl at the end of the file

**Comment by jorge07** [comment]
@Lutacon thanks for you PR. I just drop few questions ;)

File: src/UI/Http/Rest/Controller/Event/GetEventsController.php (hunk 1 above)
Comment (Lutacon) [reply src/UI/Http/Rest/Controller/Event/GetEventsController.php]: https://github.com/sensiolabs/SensioFrameworkExtraBundle/blob/master/CHANGELOG.md#52

The Route from FrameworkExtraBundle is deprecated in the new version.

File: config/packages/framework_extra.yml (hunk 2 above)
Comment (Lutacon) [reply config/packages/framework_extra.yml]: This is needed for the deprecated thing.

File: config/mapping/orm/projections/User.Query.UserView.orm.xml (hunk 4 above)
Comment (Lutacon) [reply config/mapping/orm/projections/User.Query.UserView.orm.xml]: It's not needed, my bad.

File: config/mapping/orm/domain/User.ValueObject.Auth.Credentials.orm.xml (hunk 3 above)
Comment (Lutacon) [reply config/mapping/orm/domain/User.ValueObject.Auth.Credentials.orm.xml]: It's not needed, my bad.

File: src/UI/Http/Rest/Controller/Event/GetEventsController.php (hunk 1 above)
Comment (jorge07) [reply src/UI/Http/Rest/Controller/Event/GetEventsController.php]: +1 for a separate PR for this

**Pull request closed without merging**